examples/snake/server/config/contractAddresses.ts

```diff
@@ -0,0 +1,52 @@
+import { readFileSync } from 'fs';
+import { join } from 'path';
+import { execSync } from 'child_process';
+import { Hex } from 'viem';
+
+interface BroadcastTransaction {
+  contractAddress: string;
+  contractName: string;
+}
+
+interface BroadcastFile {
+  transactions: BroadcastTransaction[];
+}
+
+function getRepoRoot(): string {
+  return execSync('git rev-parse --show-toplevel', { encoding: 'utf8' }).trim();
+}
+
+function getContractAddressFromBroadcast(contractName: string, chainId: number = 137): Hex {
```

Comment:
you dont need to use any contracts right now; As a flow is next;

user -> apps.yellow.com -> connect eoa -> create channel -> open snake app -> connect eoa in the app -> play game

app -> apps.yellow.com -> connect eoa -> create channel ->  copy private key -> use private key on the snake server